js/app.js

```diff
@@ -45,7 +42,7 @@ function buildArray1() {
   var ul = document.createElement('ul');
   article.appendChild(ul);
 
-  for (var x = 0; x < custAndCookiesPerHour.length; x++) {
+  for (var x = 0; x < firstAndPike.custAndCookiesPerHour.length; x++) {
     var li = document.createElement('li');
     li.textContent = custAndCookiesPerHour[x];
```

Comment:
custAndCookiesPerHour is part of the object. What do we need to do to access a property inside of the object?